typescript/db/src/repository/workspace.ts

```diff
@@ -24,18 +24,17 @@ export const addWorkspace: Repository["workspace"]["add"] = async (
   if (typeof user?.id !== "string") {
     throw new Error("Couldn't create workspace: No user id");
   }
-  const slug = slugify(workspace.name, { lower: true });
```

Comment:
~~As said above, I disagree with this change. The slugification should be done server side to avoid security problems.~~

Reply:
Ok nevermind got it, it's not done client side, it's done in the resolver.